aip/cloud/2510.md

```diff
@@ -28,48 +28,50 @@ different services use different project identifiers.
 is an [alias][], and behaves as such. Additionally, third-party services are
 unable to _accept_ project IDs.
 
-Externally-facing Google APIs **should** accept both project IDs and project
-numbers for incoming API requests.
-
-However, the project number is the canonical identifier, and therefore APIs
-**must** use project numbers in responses, regardless of whether they were sent
-a project ID or a project number.
-
 The rationale for this is:
 
-- Resource should always have one canonical identifier.
+- Each resource should always have one canonical identifier.
 - Because Google's privacy policy restricts the use of project IDs, both
   internally and with partners, only the project number can be the canonical
   identifier.
 - Translating from an [alias][] to a canonical identifier should be consistent
-  everywhere, including here.
+  everywhere, including project identifiers.
```

Comment:
It's not clear to me what "project identifiers" means here. It can't really mean "project ID" (which would be the obvious expansion) as then we're saying "For a project ID, you've got to use a project number instead of a project ID". Possibly "project resource names"?

Reply:
I took this to mean that the general "return the canonical form when an alias is sent" rule applies to project identifiers as a whole. It is unfortunate that "identifiers" is overloaded here.

Maybe "...including referring to projects"?

Reply:
That would definitely be better, yes.